news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,200 @@
+This rating is directly taken into account in Productboard and gives a **user score**. More about it in the second part.
+
+
+### Designing a feature
+Lastly, once you know what to focus on, and what you should work on next, it is actually time to design the feature, to define what to build.
+
+All the product development process at PrestaShop is summarized in the following chart.
+
+![Product development steps at PrestaShop](/assets/images/2020/05/product-development-steps.png)
+
+Designing a feature means defining what is the best solution to a problem, a need.
+There are many ways to solve a problem. But design is not about solving a problem, it is about solving a problem the best way possible and with elegance.
+
+<!-- Should we keep or remove the folowwing paragraph? -->
+<!-- The product team has many constraints to take into account when working on features. Some are quite obvious, such as technical feasibility, but when maintaining an open source software used in many countries, on hundreds of thousands of online stores, the coherence and the flexibility are key. Building a feature for PrestaShop means building a software for people with different culture (Right to left languages for example), different types of businesses (online retails versus subscriptions), different types of catalog (someone selling screws doesn’t have the same catalog management problematic as someone selling clothes), and so on.
+As said earlier, adding a feature to a software means making it more complex to use and to maintain. And the team has to limit this complexity. That is not an easy task, and collaboration, discussions with users are key to succeed here. That is the **User Research** phase. -->
+
+The usual process to design and build a feature is the following:
+- User Research
+- Prototyping
+- Specifications
+- Development
+- QA testing
+- Release
+
+And sometimes there are several iterations in between to reach a complete feature.
+
+Well, that is also something that the Core product team will be able to share publicly from now on thanks to Productboard. You’ll be able to see the features “In progress”, and at which step they are in the process.
+Of course, here again you’ll have the possibility to comment and share your feedback on what’s being built. Even to manifest yourself if you want to participate in prototype testing and/or  development.
+
+
+
+Overall, Productboard will improve PrestaShop’s product development transparency and facilitate contributions from the community at every step of the process. From sharing an insight, to rating a need to solve, to contributing the user research, your voice will be heard and taken directly into account. That is the goal behind this move to Productboard.
+
+
+Here is the second part, where you’ll learn more about the PrestaShop Feedback interface.
+
+
+## How to use PrestaShop Feedback
+
+### See all features and expressions of needs, follow their development
```

Comment:
'features and expressions of needs' ou 'features or expressions of needs' ?

Reply:
🤔 good question
Don't see a big difference either way.
I'll keep it like that.